integration-tests/bulkExport.test.ts

```diff
@@ -0,0 +1,168 @@
+    ) => {
+        // For each resourceType get all fileUrls
+        const resourceTypeToFileUrls: Record<string, string[]> = mapValues(
+            groupBy(outputs, 'type'),
+            (outs: ExportStatusOutput[]) => {
+                return outs.map(out => {
+                    return out.url;
+                });
+            },
+        );
+
+        // Get all resources from exported files for each resourceType
+        const resourceTypeToResourcesInExportedFiles: Record<string, any[]> = {};
+        // eslint-disable-next-line no-restricted-syntax
+        for (const [resourceType, urls] of Object.entries(resourceTypeToFileUrls)) {
+            const fileDataPromises = urls.map(url => {
+                return BulkExportTestHelper.downloadFile(url);
+            });
+            // eslint-disable-next-line no-await-in-loop
+            resourceTypeToResourcesInExportedFiles[resourceType] = (await Promise.all(fileDataPromises)).flat();
+        }
+
+        // Check that resources were exported to S3 files
+        Object.entries(resourceTypeToResourcesInExportedFiles).forEach(entry => {
+            const [resourceType, resourcesInExportedFile] = entry;
+            expect(resourcesInExportedFile).toContainEqual(resTypToResExpectedInExport[resourceType]);
+        });
+
+        // Check that resources were not exported to S3 files
+        if (Object.keys(resTypToResNotExpectedInExport).length > 0) {
+            Object.entries(resourceTypeToResourcesInExportedFiles).forEach(entry => {
+                const [resourceType, fileData] = entry;
+                expect(fileData).not.toContainEqual(resTypToResNotExpectedInExport[resourceType]);
+            });
+        }
+    };
+
+    test(
+        'Successfully export all data added to DB after currentTime',
+        async () => {
+            // BUILD
+            const oldCreatedResourceBundleResponse = await bulkExportTestHelper.sendCreateResourcesRequest();
+            const resTypToResNotExpectedInExport = getResources(oldCreatedResourceBundleResponse);
+            const currentTime = new Date();
+            const newCreatedResourceBundleResponse = await bulkExportTestHelper.sendCreateResourcesRequest();
+            const resTypToResExpectedInExport = getResources(newCreatedResourceBundleResponse);
+
+            // OPERATE
+            // Only export resources that were added after 'currentTime'
+            const statusPollUrl = await bulkExportTestHelper.startExportJob({ since: currentTime });
+            const responseBody = await bulkExportTestHelper.getExportStatus(statusPollUrl);
+
+            // CHECK
+            return checkResourceInExportedFiles(
+                responseBody.output,
+                resTypToResExpectedInExport,
+                resTypToResNotExpectedInExport,
+            );
+        },
+        FIVE_MINUTES_IN_MS,
+    );
+
+    function checkOnlyFilesSpecifiedByTypeAreExported(outputs: ExportStatusOutput[], type: string) {
```

Comment:
I don't like how this function is declared between tests. It's better to just keep `test` declarations within the `describe` block.

Also, this function is only used once. I thin you can just inline on L150 like 
```js
expect(new Set((responseBody.output as ExportStatusOutput[]).map(x => x.type))).toEqual(new Set([type]));
```

Reply:
updated to use this new one line transform instead of the function